region/norfair/west/Business Center.json

```diff
@@ -1176,88 +1353,1194 @@
+      },
+      "requires": [
+        {"shinespark": {"frames": 5}}
+      ],
+      "exitCondition": {
+        "leaveWithSpark": {}
+      },
+      "unlocksDoors": [
+        {"types": ["super"], "requires": []},
+        {"types": ["missiles", "powerbomb"], "requires": ["never"]}
+      ]
+    },
+    {
+      "link": [6, 2],
+      "name": "Carry Shinecharge (Sova Evade or Kill)",
+      "entranceCondition": {
+        "comeInShinecharged": {
+          "framesRequired": 115
+        }
+      },
+      "requires": [
+        {"or": [
+          "canShinechargeMovementTricky",
+          {"and": [
+            "canShinechargeMovementComplex",
+            "ScrewAttack"
+          ]}
+        ]}
+      ],
+      "exitCondition": {
+        "leaveShinecharged": {
+          "framesRemaining": "auto"
+        }
+      },
+      "unlocksDoors": [
+        {"types": ["super"], "requires": []},
+        {"types": ["missiles", "powerbomb"], "requires": ["never"]}
+      ],
+      "note": "If Screw Attack is unavailable, do a low spin jump to make it over the Sova without taking a hit."
+    },
+    {
+      "link": [6, 2],
+      "name": "Carry Shinecharge (Sova Damage)",
+      "entranceCondition": {
+        "comeInShinecharged": {
+          "framesRequired": 135
+        }
+      },
+      "requires": [
+        {"enemyDamage": {
+          "enemy": "Sova",
+          "type": "contact",
+          "hits": 1
+        }},
+        "canShinechargeMovementComplex"
+      ],
+      "exitCondition": {
+        "leaveShinecharged": {
+          "framesRemaining": "auto"
+        }
+      },
+      "unlocksDoors": [
+        {"types": ["super"], "requires": []},
+        {"types": ["missiles", "powerbomb"], "requires": ["never"]}
+      ]
+    },
+    {
+      "link": [6, 2],
+      "name": "Come in Shinecharging, Leave Shinecharged (Slide Off)",
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 4,
+          "openEnd": 1
+        }
+      },
+      "requires": [
+        "canShinechargeMovementTricky",
+        "canTemporaryBlue"
+      ],
+      "exitCondition": {
+        "leaveShinecharged": {
+          "framesRemaining": 85
+        }
+      },
+      "unlocksDoors": [
+        {"types": ["super"], "requires": []},
+        {"types": ["missiles", "powerbomb"], "requires": ["never"]}
+      ],
+      "note": [
+        "Slide off the ledge while gaining a shinecharge, and hold forward to clear both platforms, killing the Sova with temporary blue."
+      ]
+    },
+    {
+      "link": [6, 2],
+      "name": "Come in Shinecharging, Leave Shinecharged (Sova Kill)",
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 3,
+          "openEnd": 1
+        }
+      },
+      "requires": [
+        "canShinechargeMovementComplex",
+        {"enemyKill": {
+          "enemies": [["Sova"]],
+          "explicitWeapons": ["ScrewAttack", "Wave", "Spazer", "Plasma", "Missile", "Super"]
```

Comment:
A downback is way easier than a tiny spinjump to shot-kill the sova.

And power beam delay-shot is very consistent.  The trick is to store spark with a forward diagaonl so you start running again right away.  And downback on the way down.

Reply:
I just tried again for another hour or so. I don't know what I'm doing wrong but it's not working for me. The only times I've seen it work is if I happen to fall with just the right speed to thread the needle between the platform to the left and the Sova below, avoid bonking the platform; and when I get that it doesn't seem to matter if I shoot the Sova or not. Any chance you could make a video?